patches/bess/0009-Include-cjose-and-jansson-for-NTF.patch

```diff
@@ -0,0 +1,25 @@
+From 01f457b5d29a64b5ba434da3009c8b3200ca9a2c Mon Sep 17 00:00:00 2001
+From: Aaron Sinclair <aaron@selfienetworks.com>
+Date: Wed, 17 Jun 2020 09:59:53 -0700
+Subject: [PATCH] Include cjose and jansson for NTF
+
+---
+ core/Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/core/Makefile b/core/Makefile
+index b2921e10..6252b141 100644
+--- a/core/Makefile
++++ b/core/Makefile
+@@ -80,7 +80,7 @@ endif
+ # user requests a static build.
+ ALWAYS_DYN_LIBS := -lpthread -ldl
+ # These libraries are not supported by pkg-config.
+-ALWAYS_LIBS := -lpcap -lgflags -lnuma -lbpf
++ALWAYS_LIBS := -lpcap -lgflags -lnuma -lbpf -lcjose -ljansson -lcrypto
```

Comment:
can we try to see if having an `extra-ntf.mk` in your repo copied to `core/` would work? we currently use extra-upf.mk

```
LIBS += -lcjose -ljansson -lcrypto
```

relevant upstream snippets
https://github.com/NetSys/bess/blob/c648ea8d39c00bb67df6cdd64bb3f92b48036175/core/Makefile#L160-L165
https://github.com/NetSys/bess/blob/c648ea8d39c00bb67df6cdd64bb3f92b48036175/core/Makefile#L185

Reply:
Yes - this is the reason the build is failing at the moment; I'm only building cjose if NTF is enabled.

On that note @krsna1729 do you think it makes sense to compile cjose conditinally, or should this always go in?

Reply:
if will be conditionally since your extra-ntf.mk will be copied in only under the if block

Reply:
unresolving till we know if we can use extra-ntf.mk to as a substitute for this patch

Reply:
extra-ntf.mk will be present in your repo/core/

Reply:
@krsna1729 The sample plugin repos don't have the `core` directory so this complicates things - however we now have extra-ntf.mk in the plugin directory to take care of this so I think this is good to resolve.

Reply:
@nevelis i meant repo/modules

Reply:
but i guess they are not looking at modules/mk

Reply:
really wish we can fix the plugin model build then we can use ntf that way. Also move our modules to plugin mechanism too.
#4

Reply:
until then this is fine